test/image-server.etags.json

```diff
@@ -8,7 +8,7 @@
 	"http://localhost:13337/UP_steam_loco.jpg/:/cr=l:5%25,t:10%25,w:90%25,h:80%25/fm=f:jpeg?cache=false": "4292299306",
 	"http://localhost:13337/UP_steam_loco.jpg/:/cr=l:50,t:50,w:-100,h:-100/fm=f:jpeg?cache=false": "3539176172",
 	"http://localhost:13337/UP_steam_loco.jpg/:/cr=l:50,t:50/fm=f:jpeg?cache=false": "496358728",
-	"http://localhost:13337/UP_steam_loco.jpg/:/cr=w:50%25,h:50%25,a:tl/fm=f:jpeg?cache=false": "2536073760",
+	"http://localhost:13337/UP_steam_loco.jpg/:/cr=w:50%25,h:50%25,a:tl/fm=f:jpeg?cache=false": "3685393672",
```

Comment:
Good catch - this was caused by using `Math.round` - it was causing `tl` to crop at 1px instead of 0px - I fixed it